index.jsx

```diff
@@ -1,15 +1,27 @@
 import React, {PropTypes} from 'react';
 
-function radio(name, selectedValue, onChange) {
+function radio(kwargs) {
+
   return React.createClass({
     render: function() {
+      const name = kwargs.name;
+
+      // We only want to assign these attribute to the <input />
+      // elements if they were specified explicitly.
+      const optional = {};
+      if(kwargs.hasOwnProperty("selectedValue")) {
```

Comment:
Yeah, the more I think about it, the more I'm inclined to agree.  I'll update the PR in just a moment.